### DISCUSSION TITLE
Request for "File Server" - Delete folders still including files

### DISCUSSION THREAD

**Discussion by drcliff** [jomjol/AI-on-the-edge-device#1982] [Ideas]
Via fileserver menu actually it is not possible to delete a folder, which is not empty.
It would be great, if it may be possible to delete a folder, including all sub-files and sub folders.
Maybe a windows will popup, asking the user, if he really want to delete the directory with all sub folders and files,
if it is not empty.

I were just deleting some old log images i recently collected. It took around 10 minutes, just for manually deleting all files and folders manually.

Thank you.

**Comment by caco3** [comment]
please create a feature request.
but I don't think this has high priority!
